### DISCUSSION TITLE
Contractual issues

### DISCUSSION THREAD

**Discussion by mxyfighting** [HewlettPackard/swarm-learning#160] [Q&A]
Hello, can you answer for me? How to create your own smart contract through the swci node? The information error in the creation report is shown in the figure below. Is there a specific process? At the same time, where is the default smart contract? Can users modify the content of the default smart contract?
![7](https://user-images.githubusercontent.com/106064677/221111228-4c33d365-2e29-456e-be30-979ed353b6d0.png)

**Comment by iArpanPatel** [comment]
@mxyfighting apologies for late response, we missed seeing this question.

Creating new contract is not supported in Community version, its available only in Enterprise version. You would need to create new contract only if you are running concurrent Swarm trainings.

Users cannot modify the content inside the smart contract, as this is meant for our (Swarm) internal usage. 
What is your use case?
